### DISCUSSION TITLE
MODBUS RTD example?

### DISCUSSION THREAD

**Discussion by levancho** [Pi4J/pi4j#143] [Q&A] [answered]
is there example of reading modbus device sensor data anywhere?

**Comment by FDelporte** [comment]
What kind or type of sensor are you using? Don't think we have Pi4J implementations for this...
I found this modbus Java implementation https://sourceforge.net/projects/easymodbustcp-udp-java/#focus

**Comment by savageautomate** [comment, marked as answer]
Pi4J does not implement any MODBUS communication protocol or specific MODBUS chipsets at this time.
